samples/bluetooth/mtu_update/central/src/central_mtu_update.c

```diff
@@ -205,7 +206,11 @@ static void connected(struct bt_conn *conn, uint8_t err)
 
 	printk("Connected: %s\n", addr);
 
-	(void)mtu_exchange(conn);
+	(void)bt_conn_get_info(conn, &info);
+
+	if (info.role == BT_CONN_ROLE_CENTRAL) {
+		(void)mtu_exchange(conn);
+	}
```

Comment:
Seems superfluous to guard a function call with `BT_CONN_ROLE_CENTRAL` in a central sample.

Furthermore, both the central and peripheral is allowed to do the MTU exchange